modal/image.py

```diff
@@ -336,6 +336,24 @@ def copy(self, mount: _Mount, remote_path: Union[str, Path] = ".") -> "_Image":
             context_mount=mount,
         )
 
+    def copy(self, mount: _Mount, remote_path: Union[str, Path] = ".") -> "_Image":
+        deprecation_warning(
+            date(2023, 5, 21),
+            "`Image.copy` is deprecated in favor of `Image.add_mount`, `Image.add_local_file`,"
+            " and `Image.add_local_dir`.",
+        )
+        return self.add_mount(mount, remote_path)
+
+    def add_local_file(self, local_path: Union[str, Path], remote_path: Union[str, Path]) -> "_Image":
```

Comment:
`copy_local_file` (and similar for the others) might be slightly better so that people reaching for Dockerfile `COPY` find this